spec/order_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "order"
+require "menu"
+
+describe Order do
+  subject(:order) { described_class.new(menu) }
+
+  let(:menu) { double(:menu) }
+  let(:foods) do
+    {
+      wings: 5.00,
+      pizza: 7.00
+    }
+  end
+
+  before do
+    allow(menu).to receive(:has_food?).with(:wings).and_return(true)
+    allow(menu).to receive(:has_food?).with(:pizza).and_return(true)
+    allow(menu).to receive(:price).with(:wings).and_return(5.00)
+    allow(menu).to receive(:price).with(:pizza).and_return(7.00)
+  end
+
+  describe "#add_to_order" do
+    it "it enables several dishes to be selected from the menu" do
```

Comment:
Small thing but don't forget if a test fails it says "class (eg. Order)" and then goes into your describe string so you don't need to say "IT enables" because that will come out as "Order it enables"